.github/workflows/test-performance-android.yml

```diff
@@ -19,7 +19,7 @@ on:
         required: true
         default: 'master'
   schedule:
-    - cron: '0 6 * * *'
```

Comment:
Avoid starting both e2e jobs at the same time. Will change to manual start after seeing that it works.